.cirrus.yml

```diff
@@ -48,7 +48,7 @@ image_builder_task:
     auto_cancellation: $CI != "true"
     stateful: true
     timeout_in: 50m
-    container:
+    container: &ibi_container
```

Comment:
`&ibi_container` is never used again. Is this a leftover from some previous iteration?

Reply:
Oops, meant to comment on this.  Yes, it's a leftover, I have a staged change to remove it.